AccessTransformers: Prevent present but empty transformer file from breaking builds

```
********** BUILD COMMAND STARTED **********
Running: C:\Program Files\Unreal Engine - CSS\Engine\Binaries\ThirdParty\DotNet\6.0.302\windows\dotnet.exe "C:\Program Files\Unreal Engine - CSS\Engine\Binaries\DotNET\UnrealBuildTool\UnrealBuildTool.dll" -Target="FactoryGameSteam Win64 Shipping -Project=T:\Git\SatisfactoryModLoader\FactoryGame.uproject -Manifest=T:\Git\SatisfactoryModLoader\Intermediate\Build\Manifest-1-FactoryGameSteam-Win64-Shipping.xml  -remoteini=\"T:\Git\SatisfactoryModLoader\"  " -Target="FactoryServer Win64 Shipping -Project=T:\Git\SatisfactoryModLoader\FactoryGame.uproject -Manifest=T:\Git\SatisfactoryModLoader\Intermediate\Build\Manifest-2-FactoryServer-Win64-Shipping.xml  -remoteini=\"T:\Git\SatisfactoryModLoader\"  " -log="C:\Users\user\AppData\Roaming\Unreal Engine\AutomationTool\Logs\C+Program+Files+Unreal+Engine+-+CSS\UBT-.txt"
Log file: C:\Users\user\AppData\Roaming\Unreal Engine\AutomationTool\Logs\C+Program+Files+Unreal+Engine+-+CSS\UBT-.txt
Using 'git status' to determine working set for adaptive non-unity build (T:\Git\SatisfactoryModLoader).
Creating makefile for FactoryGameSteam (command line arguments changed)
FactoryGameSteam: Building with command line overrides: ChecksInShipping: True (default: false); LinkType: Modular (default: Modular)
[FactoryGameSteam] FactoryGame: IsPublicBuild: 1 WithCheats: 0
Parsing headers for FactoryGameSteam
  Running Internal UnrealHeaderTool T:\Git\SatisfactoryModLoader\FactoryGame.uproject T:\Git\SatisfactoryModLoader\Intermediate\Build\Win64\FactoryGameSteam\Shipping\FactoryGameSteam.uhtmanifest -WarningsAsErrors -installed
Total of 0 written
C:\Program Files\Unreal Engine - CSS\Engine\Source\UnknownSource(1): Internal Compiler Error: System.NullReferenceException - Object reference not set to an instance of an object.
Total execution time: 5.30 seconds
Took 5,41s to run dotnet.exe, ExitCode=6
Cleaning Stage Directory: T:\Git\SatisfactoryModLoader\Mods\SML\Saved\StagedBuilds\Windows
Cleaning Stage Directory: T:\Git\SatisfactoryModLoader\Mods\SML\Saved\StagedBuilds\WindowsServer
UnrealBuildTool failed. See log for more details. (C:\Users\user\AppData\Roaming\Unreal Engine\AutomationTool\Logs\C+Program+Files+Unreal+Engine+-+CSS\UBT-.txt)
AutomationTool executed for 0h 0m 8s
AutomationTool exiting with ExitCode=6 (6)
BUILD FAILED
Packaging failed with ExitCode=6
```

https://discord.com/channels/555424930502541343/1036634533077979146/1300408607308775516

https://discord.com/channels/555424930502541343/1036634533077979146/1300474863844327514

https://discord.com/channels/555424930502541343/1036634533077979146/1300512962456584253

See more info in #307  (I opened this without realizing Jarno already had)

--- a/Mods/AccessTransformers/Source/AccessTransformers.ubtplugin/Types/AccessTransformers.cs
+++ b/Mods/AccessTransformers/Source/AccessTransformers.ubtplugin/Types/AccessTransformers.cs
@@ -110,7 +110,7 @@ public static class AccessTransformers
             {
                 var accessTransformers = new ConfigFile(accessTransformersFile, ConfigLineAction.Add);
                 accessTransformers.TryGetSection("AccessTransformers", out var section);
-                section!.Lines.ForEach((line) =>
+                section?.Lines.ForEach((line) =>
                 {
                     switch (line.Key)
                     {

